### PR TITLE
Add samples

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by odscjames** [comment]
Also fixed one issue I found in examples that was an easy fix, see last commit.
* Before CI: https://github.com/openreferral/specification/runs/3241719146?check_suite_focus=true
* After CI: https://github.com/openreferral/specification/runs/3241727292?check_suite_focus=true

This is not perfect in that when new examples are added then a new block must be added to the CI config file. But fixing that will be difficult, so let's get this in to start with.

File: README.md [hunk 1]
```diff
@@ -69,3 +69,14 @@ pip install sphinx-autobuild
...
+## Examples
+
+Examples are included with the documentation and these should match the current datapackage.json. To test:
+
+    pip install frictionless
+    python examples/make_datapackages.py
+    frictionless validate examples/all_resources/datapackage.json
+    frictionless validate examples/simple/datapackage.json
+
+Everytime a new example is added, please add a line above and in `.github/workflows/test_examples.yml`.
```
Comment (robredpath) [inline README.md:82]: @odscjames It's not clear to me what "an example" is. Is it a new CSV file? A new directory in `/examples`? Both of these things? It'd be good to be more explicit about this.

**Comment by robredpath** [comment]
Thanks @odscjames . This is looking good, but we're getting to the point where I think we need input from @greggish on the content of this, on two points:
* Is this "example data" or "sample data", and does it matter? 
* The actual content of the sample data - especially around things like where the original creator chose to use UUIDs where they're not technically required, and the confusion that might lead to. 

If that's awkward (as that latter point might be a fairly nuanced conversation), can we get this on to a development branch, which we merge in when the appropriate time comes? Does that work for you, in terms of workflow?

File: README.md (hunk 1 above)
Comment (odscjames) [reply README.md]: Expanded

**Comment by odscjames** [comment]
https://dictionary.cambridge.org Example says "a way of helping someone to understand something by showing them how it is used" whilst sample says "a small amount of something that shows you what the rest is or should be like" and I think in the docs we are more focussed towards the learning and understanding side, so maybe Example? But also I don't have strong views.

Maybe one way to think about this is that as soon as we are happy with the sample/example data and it actually matches the schema we should add links to the HSDS section of the docs. Maybe thinking about how that information will be presented will help us work out which language to use now.

In terms of the content, I've copied almost totally from an existing repository (as I've made clear in the commit message for it, so that when people are working out the history later I don't get the blame/credit) so I don't know if these issues need to be revolved now or if we can merge and resolve as we fix up the sample/example data anyway.

But if delay is wanted, I don't see any problems with leaving this as a open Pull Request for a bit while those things are checked; I don't think this will conflict with anything else that is coming up.

**Comment by greggish** [comment]
I don't have a strong opinion on `sample` vs `example`, I guess I'd lean slightly toward `example` although if we switch to that we'll probably want to edit documentation to be consistent. 

As for the non-required UUIDs... hard to say. Are they not only nonrequired, but also not useful? If they're not required but having them there wouldn't be potentially useful, then I don't think they need to come out. If it's an instance in which IDs were added to entities that really don't need IDs, then it might make sense to take them out.

I guess this brings up a different issue, which is: would it be worth assembling a minimally-viable data example, and also a 'complete' data example??

**Comment by odscjames** [comment]
Talked to @robredpath - We'll go with this PR as is (So asking for review) and we'll open 2 new issues:

* look at the UUID issue
* the language of "samples" & "examples" is used inconsistently in the docs; there is a samples page but it's not linked from menus: fix both these at once

**Review by robredpath — APPROVED** [review]

**Pull request merged**